webapp/src/components/Portfolio/PortfolioPositions.tsx

```diff
@@ -49,19 +49,54 @@ const SectionPlaceholderText = styled(SecondaryText)`
 
 const PositionLink = styled(BaseLink)`
   width: 100%;
-  margin-top: 16px;
+  margin-top: 24px;
+`;
+
+const PositionContainer = styled.div<{ color: string }>`
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+  border-radius: ${theme.border.radius};
+  border: 2px ${theme.border.style} #00000000;
 
-  &:first-child {
-    margin-top: 24px;
+  &:hover {
+    box-shadow: ${(props) => props.color}3D 8px 16px 80px;
+    border: 2px ${theme.border.style} ${(props) => props.color};
   }
 `;
 
-const PositionContainer = styled.div`
+const PositionMainContainer = styled.div`
+  display: flex;
+  align-items: center;
   width: 100%;
-  border: ${theme.border.width} ${theme.border.style} ${colors.border};
-  border-radius: ${theme.border.radius};
   padding: 16px;
+  background: ${colors.background};
+  border-radius: ${theme.border.radius};
+  border: ${theme.border.width} ${theme.border.style} ${colors.border};
+  z-index: 2;
+`;
+
+const PositionStakedContainer = styled(PositionMainContainer)`
+  margin-top: -16px;
+  padding-top: calc(16px + 16px);
+  background: ${colors.backgroundLighter};
+  z-index: 1;
+`;
+
+const LogoContainer = styled.div<{ color: string }>`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-right: 8px;
+  width: 40px;
+  height: 40px;
+  border-radius: 100px;
+  background: ${(props) => props.color}29;
```

Comment:
Is this a typo? There is a '29' at the end

Reply:
Oh I see that this is a common pattern used. Does the last 2 hex set the alpha for the color?

Reply:
Yup. So each asset has their own color, and the rest of the color variation is a product of adding alpha to the base color